iphone/Classes/TiUITextArea.m

```diff
@@ -218,6 +233,11 @@ - (void)setEnabled_:(id)value
   [(UITextView *)[self textWidgetView] setEditable:_trulyEnabled];
 }
 
+- (void)setEnableCopy_:(id)value
```

Comment:
You can move this API inside TiUITextWidgetProxy.

Reply:
But the implementation is different between `TiUITextArea` and `TiUITextField`. This function operates on `TiUITextViewImpl` and the other function operates on `TiTextField`.